source/Junkership.js

```diff
@@ -3,43 +3,62 @@ var Keyb = require("keyb")
 
 import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
+import Score from "./Score.js"
 
 var junkerTex = Pixi.Texture.fromImage(require("./images/blue-starship.png"))
 
 export default class Junkership extends Pixi.Sprite {
     constructor() {
         super(junkerTex)
-        this.speed = 60
-        this.score = 0
         game.playerCount++
+        this.speed = 60
+        this.score = new Score()
     }
     update(delta) {
-        if(Keyb.isDown("<up>") && this.position.y > 0) {
-            this.position.y -= this.speed * delta
-        }
-        if(Keyb.isDown("<down>") &&
-            (this.position.y + this.height) < Reference.GAME_HEIGHT) {
-            this.position.y += this.speed * delta
+        var relativeSpeed = this.speed * delta
+        if(Keyb.isDown("<up>")) {
+            this.move(-relativeSpeed, "y")
+        } else if(Keyb.isDown("<down>")) {
+            this.move(relativeSpeed, "y")
```

Comment:
I actually think moving in the direction last pressed, but was not sure how to implement that. I simply changed it to the way above because when I was looking into this issue, this is how I seen others implement this. I'll probably look further into this because as I said I don't really care for my solution, but didn't like the idea of not moving when you're holding a button either. It felt counterintuitive as well.